lib/ansible/modules/system/seboolean.py

```diff
@@ -79,7 +79,12 @@ def has_boolean_value(module, name):
         rc, bools = selinux.security_get_boolean_names()
     except OSError:
         module.fail_json(msg="Failed to get list of boolean names")
-    if to_bytes(name) in bools:
+    # work around for selinux who changed its API, see
+    # https://github.com/ansible/ansible/issues/25651
+    if len(bools) > 0:
+        if isinstance(bools[1], type(b'')):
```

Comment:
* Should this be ```bools[0]```?
* You can use six.binary_type instead of calling type() as well.  Then things will look like this:

``` python
from ansible.module_utils.six import binary_type
if len(bools) > 0:
    if isinstance(bools[0], binary_type):
```